server/game/cards/01_SOR/units/BodhiRook.ts

```diff
@@ -0,0 +1,33 @@
+import AbilityHelper from '../../../AbilityHelper';
+import { NonLeaderUnitCard } from '../../../core/card/NonLeaderUnitCard';
+import { RelativePlayer, WildcardCardType } from '../../../core/Constants';
+
+export default class BodhiRook extends NonLeaderUnitCard {
+    protected override getImplementationId() {
+        return {
+            id: '7257556541',
+            internalName: 'bodhi-rook#imperial-defector'
+        };
+    }
+
+    public override setupCardAbilities() {
+        this.addWhenPlayedAbility({
+            title: 'Look at an opponent\'s hand and discard a non-unit card from it.',
+            immediateEffect: AbilityHelper.immediateEffects.sequential([
+                AbilityHelper.immediateEffects.lookAt((context) => ({
+                    target: context.player.opponent.hand.sort((a, b) => a.title.localeCompare(b.title)),
+                    sendChatMessage: true
```

Comment:
Looking back, I don't know why Viper Probe Droid was formatted with the ability like this. You can drop `sendChatMessage` since the default value is true, and the targets don't need to be sorted.

If you wouldn't mind also updated Probe Droid to match since people will probably refer to it as an example a lot, that'd be really helpful

Reply:
Updated the setHand function to not put the cards back in reverse.  Removed the sort from Viper and Bodhi.  Updated their unit tests.  Also updated the Chimera's unit test for the now flipped order.